Makefile

```diff
@@ -52,8 +52,10 @@ install: dwl
 	mkdir -p $(DESTDIR)$(MANDIR)/man1
 	cp -f dwl.1 $(DESTDIR)$(MANDIR)/man1
 	chmod 644 $(DESTDIR)$(MANDIR)/man1/dwl.1
+	cp -f dwl.desktop $(DESKTOP)/dwl.desktop
```

Comment:
This should be copied to `$(DESTDIR)$(PREFIX)/share/wayland-sessions/dwl.desktop`